data/remnant/remnant news.txt

```diff
@@ -0,0 +1,307 @@
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+
+
+phrase "00-07"
+	word
+		"00"
+		"01"
+		"02"
+		"03"
+		"04"
+		"05"
+		"06"
+		"07"
+
+
+
+phrase "0-F"
+	word
+		"0"
+		"1"
+		"2"
+		"3"
+		"4"
+		"5"
+		"6"
+		"7"
+		"8"
+		"9"
+		"A"
+		"B"
+		"C"
+		"D"
+		"E"
+		"F"
+
+
+
+phrase "00-1F"
+	word
+		"0"
+		"1"
+	phrase
+		"0-F"
+
+
+
+phrase "A group of"
+	word
+		"A "
+	word
+		"pair "
+		"couple "
+		"pack "
+		"squad "
+		"team "
+		"family "
+		"group "
+		"gathering "
+		"small group "
+		"medium group "
+		"large group "
+	word
+		"of "
+
+
+
+phrase "Indeterminate"
+	word
+		"A few "
+		"Several "
+		"Some "
+		"Lots of "
+		"Many "
+
+
+
+phrase "Amount of Remnant"
+	phrase
+		"A group of"
+		"Indeterminate"
+	word
+		"Remnant"
+
+
+news "remnant security advisory"
+	location
+		government "Remnant"
+	name
+		word
+			"System Security"
+			"Local"
+			"System"
+			"Defense"
+		word
+			" "	
+		word
+			"Advisory"
+			"Status"
+			"Threats"
+	message
+		word
+			`"`
+		word
+			"Incoming Korath Fleet in "
+		word
+			"${00-07} days"
+		word
+			" "
+		phrase
+			"00-1F"
+		word
+			" hours. Threat Sierra-"
+		word
+			"2"
+			"3"
+		phrase
+			"0-F"
+		word
+			"."
+		word
+			`"`
+	message
+		word
+			`"`
+		word
+			"Incoming Korath Ship in "
+		word
+			"${00-07} days"
+		word
+			" "
+		phrase
+			"00-1F"
+		word
+			" hours. Threat Sierra-"
+		word
+			"4"
+			"5"
+		phrase
+			"0-F"
+		word
+			"."
+		word
+			`"`
+	message
+		word
+			`"`
+		word
+			"Patrol"
+			"Salvage"
+			"Sentry"
+			"Attack"
+			"Reconnaissance"
+			"Escort"
+			"Decoy"
+		word
+			" "
+		word
+			"Flight"
+			"Talon"
+			"Wing"
+		word
+			" "
+		phrase
+			"0-F"
+		phrase
+			"0-F"
+		phrase
+			"0-F"
+		word
+			" "
+		word
+			"prepare for"
+			"ready for"
+			"standby for"
+			"hold for"
+		word
+			" "
+		word
+			"departure"
+			"launch"
+			"combat"
+		word
+			"."
+		word
+			`"`
+	message
+		word
+			`"`
+		word
+			"Salvage ships report to orbit for incoming Korath ships."
+			"No change in current threat level."
+			"Maintain normal security status."
+			"All departing ships advised to have weapon systems hot."
+			"All departing ships advised to expect hostiles in the vicinity."
+			"Reminder to update your IFF settings and verify keys."
+			"Reminder to verify functionality of selective IFF broadcast every startup."
+		word
+			`"`
+
+
+
+news "remnant incomprehensible"
+	location
+		government "Remnant"
+	name
+		word
+			"Spaceport"
+			"Loading dock"
+			"Cafeteria"
+			"Landing pad"
+			"Landing dock"
+			"Spaceport alley"
+			"Spaceport hallway"
+		word
+			" scene"
```

Comment:
It seems odd to be to describe this as a "scene." If the name of a message isn't a person, then I'd expect it to be a location without it needing to say that.
```suggestion
```

Reply:
All these are just straight up "what the player sees in the named location."